src/app/pages/dynamic-route/dynamic-route.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { Component } from 'react'
+
+const MyDynamicComponent = React.lazy(() => import(/* webpackChunkName: "dynamic-component" */ "./components/dynamic-component"));
```

Comment:
One chunk per route is enogth. I see that in your code you have 3 chunks.
bundle.js
dynamic-route.bundle.js
dynamic.component.bundle.js

Ideally the dynamic-route.bundle.js should have all the code including component. code. 
I thin this import for a component can be done in stadard way. no need to have dynamic import here becasue it is creating andditional bundle.

Reply:
fixed